doc/input-output-reference/src/overview/group-design-objects.tex

```diff
@@ -1096,6 +1096,8 @@ \subsubsection{Inputs}\label{inputs-4-008}
 
 This alpha field specifies the name of a DesignSpecification:OutdoorAir object which specifies the design outdoor air flow rate for the zone.
 
+When a choice of IndoorAirQualityProcedure is entered in the Outdoor Air Method field of the DesignSpecification:OutdoorAir object, the design outdoor airflow rate is calculated based on the choice of Sum in the same field.  
+
```

Comment:
What if the method is ProportionalControlBasedOnDesignOccupancy or ProportionalControlBasedonOccupancySchedule?